.github/pull_request_template.md

```diff
@@ -16,9 +16,10 @@ Expand the checklist which is relevant for this PR.
 <p>
 
 - [ ] Issue acceptance criteria met
+- [ ] Secrets are annotated with `airbyte_secret` in output spec
 - [ ] Unit & integration tests added as appropriate (and are passing)
     * Community members: please provide proof of this succeeding locally e.g: screenshot or copy-paste acceptance test output. To run acceptance tests for a Python connector, follow instructions in the README. For java connectors run `./gradlew :airbyte-integrations:connectors:<name>:integrationTest`.
-- [ ] `/test` command documented [here]() is passing. 
+- [ ] `/test` command documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
```

Comment:
Not worth putting directly an example here, so people don't actually click to go on the doc, read, and come back?

```suggestion
- [ ] `/test connector=connectors/<name>` command as documented [here](https://docs.airbyte.io/contributing-to-airbyte/building-new-connector#updating-an-existing-connector) is passing. 
```

Reply:
good idea